doc/source/whatsnew/v0.24.0.txt

```diff
@@ -755,6 +755,7 @@ I/O
 
 - :func:`read_html()` no longer ignores all-whitespace ``<tr>`` within ``<thead>`` when considering the ``skiprows`` and ``header`` arguments. Previously, users had to decrease their ``header`` and ``skiprows`` values on such tables to work around the issue. (:issue:`21641`)
 - :func:`read_excel()` will correctly show the deprecation warning for previously deprecated ``sheetname`` (:issue:`17994`)
+- :func:`read_csv()` and func:`read_table()` will throw UnicodeEncodeError and not coredump on badly encoded strings (:issue:`22748`)
```

Comment:
double backticks on the exeption

Reply:
@jreback done

Reply:
@troels : @jreback requested double-ticks, not single.  I quickly fixed that for you.

Reply:
Ah, thanks @gfyoung !